### ISSUE TITLE
hnote fails to render in txt (ASCII Art)

### ISSUE THREAD

**Issue by bdebruyn** [plantuml/plantuml#1201]
Simple example:
```
@startuml
Bob->Alice : Hello!
hnote over Alice : Answer
@enduml
```
Using the online PlantUML generator (Online Server) and clicking on `ASCII Art` the following errors are reported:
```
java.lang.UnsupportedOperationException: NOTE_HEXAGONAL
	at net.sourceforge.plantuml.asciiart.TextSkin.createComponentNote(TextSkin.java:81)
	at net.sourceforge.plantuml.sequencediagram.graphic.DrawableSetInitializer.createNoteBox(DrawableSetInitializer.java:467)
	at net.sourceforge.plantuml.sequencediagram.graphic.DrawableSetInitializer.prepareNote(DrawableSetInitializer.java:437)
	at net.sourceforge.plantuml.sequencediagram.graphic.DrawableSetInitializer.createDrawableSet(DrawableSetInitializer.java:174)
	at net.sourceforge.plantuml.sequencediagram.graphic.SequenceDiagramTxtMaker.<init>(SequenceDiagramTxtMaker.java:97)
	at net.sourceforge.plantuml.sequencediagram.SequenceDiagram.getSequenceDiagramPngMaker(SequenceDiagram.java:258)
	at net.sourceforge.plantuml.sequencediagram.SequenceDiagram.exportDiagramInternal(SequenceDiagram.java:281)
	at net.sourceforge.plantuml.UmlDiagram.exportDiagramNow(UmlDiagram.java:135)
	at net.sourceforge.plantuml.AbstractPSystem.exportDiagram(AbstractPSystem.java:179)
	at net.sourceforge.plantuml.servlet.DiagramResponse.sendDiagram(DiagramResponse.java:159)
	at net.sourceforge.plantuml.servlet.UmlDiagramService.doGet(UmlDiagramService.java:98)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:687)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:717)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1644)
	at org.eclipse.jetty.websocket.server.WebSocketUpgradeFilter.doFilter(WebSocketUpgradeFilter.java:164)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1615)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:550)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:568)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:221)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1110)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:479)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:183)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1044)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:199)
	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:109)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
	at org.eclipse.jetty.server.Server.handle(Server.java:459)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:281)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:232)
	at org.eclipse.jetty.io.AbstractConnection$1.run(AbstractConnection.java:505)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:607)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:536)
	at java.base/java.lang.Thread.run(Thread.java:832)


An error has occured : java.lang.UnsupportedOperationException: NOTE_HEXAGONAL
Y'en a qu'ont essaye, ils ont eu des problemes
 
Diagram size: 4 lines / 68 characters.
 
PlantUML (1.2022.14beta5) cannot parse result from dot/GraphViz.
 
Please go to https://plantuml.com/graphviz-dot to check your GraphViz version.
 
Java Runtime: Java(TM) SE Runtime Environment
JVM: Java HotSpot(TM) 64-Bit Server VM
Default Encoding: UTF-8
Language: en
Country: US
 
PLANTUML_LIMIT_SIZE: 4000
 
This may be caused by :
 - a bug in PlantUML
 - a problem in GraphViz
 
You should send this diagram and this image to plantuml@gmail.com or
post to https://plantuml.com/qa to solve this issue.
You can try to turn around this issue by simplifing your diagram.
 

```

**Comment by The-Lum** [comment]
FYI: seems to be corrected with last update: [snapshot (~v1.2022.14beta7)](https://github.com/plantuml/plantuml/releases/tag/snapshot), with:
- https://github.com/plantuml/plantuml/commit/3cf5e15bb47829e74aa24c7a5ac2198f8dcb8d87#diff-e652f5db3b345e08bea370eda3f0f70a919e9cc94b74a1866c095849986cfa19

Thanks to PlantUML team.

**Comment by The-Lum** [comment]
Hi all,

_[This is an Issue Review]_
This is now fixed.
- Thanks @bdebruyn for the report.
- Thanks PlantUML team for your work.
	
Regards.
